codeship_yaml/main.py

```diff
@@ -77,17 +79,21 @@ def color_message(message, color=None):
     elif color == 'red':
         color = '\033[91m'
     else:
-        return message
+        color = None
```

Comment:
This will print `None` in the `print` statement. I think we have to provide an empty string. Then we may also add `stop_colors = ''` in order to make the print statement without any codes.